lib/pages/settings_dashboard/settings_profile/settings_profile.dart

```diff
@@ -128,21 +135,27 @@ class SettingsProfileController extends State<SettingsProfile>
   }
 
   void _handleRemoveAvatarAction() async {
-    if ((assetEntity != null || filePickerResult != null) &&
-        currentProfile?.avatarUrl == null) {
+    if (assetEntity != null || filePickerResult != null) {
+      isEditedProfileNotifier.toggle();
+    }
+    if (currentProfile?.avatarUrl == null) {
       _clearImageInLocal();
-      return;
+      settingsProfileUIState.value = Right<Failure, Success>(
+        GetProfileUIStateSuccess(
+          currentProfile!,
+        ),
+      );
+    } else {
+      TwakeLoadingDialog.showLoadingDialog(context);
+      final newProfile = Profile(
+        userId: client.userID!,
+        displayName: displayNameEditingController.text,
```

Comment:
why `displayNameEditingController.text` and not client display name ? IMO remove avatar shouldn't act as a change display name button too

Reply:
I updated it